packages/cfd/src/features/hooks/useCurrentList.ts

```diff
@@ -0,0 +1,51 @@
+import { useStore } from '@deriv/stores';
+import { useMT5AccountsList, useDxtradeAccountsList, useCtraderAccountsList } from '@deriv/api';
+import { getAccountListKey } from '@deriv/shared';
+
+type MT5AccountsList = NonNullable<ReturnType<typeof useMT5AccountsList>['data']>[number];
+type DxtradeAccountsList = NonNullable<ReturnType<typeof useDxtradeAccountsList>['data']>[number];
+type CtraderAccountsList = NonNullable<ReturnType<typeof useCtraderAccountsList>['data']>[number];
+
+type TCurrentList = {
+    [key: string]: MT5AccountsList | DxtradeAccountsList | CtraderAccountsList;
+};
+
+/* 
+    This hook is used to get the existing_accounts_data
+*/
+const useCurrentList = () => {
+    const current_list: TCurrentList = {};
+    const { traders_hub } = useStore();
```

Comment:
Can we avoid this ?

Reply:
We have handled this for wallet in useWalletAccountsList. We have `is_malta_wallet` flag. You can add this same flag in `useAccountsList` and it will be exposed to useActiveTradingAccount.

Reply:
`show_eu_related_content` is based on the `content_flag` and not `is_eu`. These 2 are not the same.

Reply:
Hrmm we dont need to use that. Maybe another PR